app/services/deploy_service.rb

```diff
@@ -32,5 +37,12 @@ def self.create_github_pull_request(deploy_strategy)
       # PR already exists
     end    
   end
+
+  def self.post_slack_notification()
+    slack = Slack::Web::Client.new
+    slack.chat_postMessage(channel: '#dev', message: "<#{pr.html_url}|Deploy PR> ready for #{repo}", as_user: true)
```

Comment:
Possible bug: I don't think `repo` is defined in this method's scope.

Also, I don't think we should hard-code `#dev` here. Replacing it with `deploy_strategy.arguments['slack_channel']` (also not defined in this scope 😄) would remain more generic _and_ allow different projects to notify different channels if desired.